gatsby/gatsby-browser.js

```diff
@@ -17,6 +17,19 @@ require("tocbot/dist/tocbot.min.js")
 require("./src/styles/global.scss")
 
 //Segment
+const previousURL = null // define constant for prevLocation in PreRouting to be mapped to
+const newURL = null // Define constant for for location in PreRouting to be mapped to, currently unused.
```

Comment:
@RJHorningCro I defined `newURL` in case we want to use that value instead of `window.location.href` for `url` in`window.analytics.page`.